src/content/eth2/merge/index.md

```diff
@@ -44,24 +44,27 @@ Plus many more.
 
 [More on Ethereum1.x](/learn/#eth-1x)
 
-These improvements all have a place in Eth2 so it’s likely that their progress may affect the timing of the docking. -->
+These improvements all have a place in Eth2 so it’s likely that their progress may affect the timing of the merge. -->
 
 ## After the merge {#after-the-merge}
 
-This will signal the end of proof-of-work for Ethereum and start the era of a more sustainable, eco-friendly Ethereum. At this point Ethereum will have the scale, security and sustainability outlined in its [Eth2 vision](/eth2/vision/).
+This will signal the end of proof-of-work for Ethereum and start the era of a more sustainable, eco-friendly Ethereum. 
+!!!!!!! At this point Ethereum will have the scale, security and sustainability outlined in its [Eth2 vision](/eth2/vision/).
 
 ## Relationship between upgrades {#relationship-between-upgrades}
 
 The Eth2 upgrades are all somewhat interrelated. So let’s recap how the merge relates to the other upgrades.
 
 ### The merge and the Beacon Chain {#docking-and-beacon-chain}
 
-Once the merge happens, stakers will be assigned to validate the Ethereum mainnet. Just like with the shard chains. [Mining](/developers/docs/consensus-mechanisms/pow/mining/) will no longer be required so miners will likely invest their earnings into staking in the new proof-of-stake system.
+Once the merge happens, stakers will be assigned to validate the Ethereum mainnet. [Mining](/developers/docs/consensus-mechanisms/pow/mining/) will no longer be required so miners will likely invest their earnings into staking in the new proof-of-stake system.
```

Comment:
should it be "the Ethereum mainnet" or just "Ethereum mainnet"?

Reply:
I vote "Ethereum mainnet"